### ISSUE TITLE
Support A11Y

### ISSUE THREAD

**Issue by lubov15911** [lubov15911/React-application#6]
There is a list with clickable <li> element. This is wrong according to a11y.

What should be done: review the whole project and fix all semantic/a11y errors.
